README.md

```diff
@@ -10,104 +10,157 @@
-    * __Subscribe to a respository__ - Use `/github subscribe` to subscribe a Mattermost channel to receive notifications for new pull requests, issues, branch creation and more in a GitHub repository
-        - For instance, to post notifications for issues, issue comments, and pull requests matching the label `Help Wanted` from `mattermost/mattermost-server`, use
-
-          ```
-          /github subscribe mattermost/mattermost-server issues,pulls,issue_comments,label:"Help Wanted"
-          ```
-        - The following flags are supported:
-           - `--exclude-org-member`: events triggered by organization members will not be delivered. It will be locked to the organization
-            provided in the plugin configuration and it will only work for users whose membership is public. Note that organization members and collaborators
-            are not the same.
-    * __Get to do items__ - Use `/github todo` to get an ephemeral message with items to do in GitHub, including a list of unread messages and pull requests awaiting your review
-    * __Update settings__ - Use `/github settings` to update your settings for notifications and daily reminders
-    * __And more!__ - Run `/github help` to see what else the slash command can do
-
-## Installation
+
+## Before You Start
+
+This guide assumes that you have a GitHub account and that you're a Mattermost System Admin.
+
+If you’re running Mattermost v5.11 or earlier, first download the latest release of this plugin from the [releases page of this GitHub repository](https://github.com/mattermost/mattermost-plugin-github/releases) and upload it to your Mattermost instance [following this documentation](https://docs.mattermost.com/administration/plugins.html#plugin-uploads).
+
+## Check your version
 
 In Mattermost 5.12 and later, the GitHub plugin is pre-packaged and no steps are required for installation. You can go directly to [Configuration](#configuration).
 
 In Mattermost 5.11 and earlier, follow these steps:
-1. Go to https://github.com/mattermost/mattermost-plugin-github/releases to download the latest release file in `zip` or `tar.gz` format.
+1. Go to https://github.com/mattermost/mattermost-plugin-github/releases to download the latest release file in zip or tar.gz format.
 2. Upload the file through **System Console > Plugins > Management**, or manually upload it to the Mattermost server under plugin directory. See [documentation](https://docs.mattermost.com/administration/plugins.html#set-up-guide) for more details.
+3. Once complete, follow the steps provided in the Configuration section.
 
 ## Configuration
 
-__If you are using GitHub Enterprise, replace all GitHub links below with your GitHub Enterprise URL__
+Configuration is started in GitHub and completed in Mattermost. 
 
-### Step 1: Register an OAuth application in GitHub
+**Note:** If you're using GitHub Enterprise, replace all GitHub links below with your GitHub Enterprise URL.
```

Comment:
@hanzei, what are your thoughts around having an EE version of this as well in GitBook?

Reply:
Fair point, I'm on board with that.